test/v1/TokenTestUtils.js

```diff
@@ -66,47 +65,23 @@ const implSlot =
 // set to true to enable verbose logging in the tests
 const debugLogging = false;
 
-function calculateFeeAmount(amount, fee, feeBase) {
-  return Math.floor((fee / feeBase) * amount);
-}
-
-function checkMinterConfiguredEvent(
```

Comment:
Where did these tests go and/or why were they removed?

Reply:
they were moved to test/v1/helpers/tokenTest.js by this test file restructuring: https://github.com/centrehq/centre-tokens/commit/4c42934a9e30da94fba65eb1da7082c4857450ed#diff-186b80475894375e2ff34eaa1dacf77fff18e70a96463064e5bfe21c55db1560

Reply:
example of a test on master using the new test file: https://github.com/centrehq/centre-tokens/blob/b5aba6ce7b5a867fda355a54a8449818b313bf1a/test/v1/events.test.js#L49.

So the utils in this class were unused